src/runtime_src/core/pcie/tools/cloud-daemon/azure/azure.cpp

```diff
@@ -178,10 +178,23 @@ static void azureHotResetAsync(size_t index)
  */
 int azureHotReset(size_t index, int *resp)
 {
-    //tell xocl don't try to restore anything since we are going
-    //to do hotplug in wireserver
-    *resp = -ESHUTDOWN;
-    nouse = std::async(std::launch::async, &azureHotResetAsync, index);
+    /*
+     * Tell xocl don't try to restore anything since we are going
+     * to do hotplug in wireserver
```

Comment:
You meant "hotplug in VM" instead of "hotplug in wireserver"?

Reply:
I meant 'hotplug initiated by wireserver'. In case the cmd is sent to wireserver, we need to return a 'SHUTDOWN' error code to tell xocl we are going to shutdown -- driver will be totally gone and be back, like a shutdown&reboot.